packages/embedded-wallet/src/ui/dialog/Dialog.tsx

```diff
@@ -55,9 +55,9 @@ const Dialog: React.FC<DialogProps> = ({
               <span className={css.title}>{title}</span>
             </h1>
           ) : (
-            {}
+            []
```

Comment:
I recomend that, to the else values of if expressions, where you don't want to show anything, that you use false or null. Because of [this](https://reactjs.org/docs/jsx-in-depth.html#booleans-null-and-undefined-are-ignored).

Reply:
Thanks for suggesting this! I've been bothered by it for quite a while :P

Reply:
I'll spin up this change in a separate PR to apply it across the entire package.